templates/taxi/driver_confirm_delete.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Delete driver</h1>
+  <form action="" method="post">
+    {% csrf_token %}
+
+    <input type="submit" value="Yes" class="btn btn-danger">
```

Comment:
Consider adding a 'Cancel' button or link to allow users to navigate away from the deletion page if they change their mind. This can improve user experience by providing an easy way to cancel the action.